### ISSUE TITLE
Unable to connect to simple game server

### ISSUE THREAD

**Issue by emrys90** [googleforgames/agones#2471]
Before I start integrating my own game servers with Agones, I wanted to verify everything works correctly. I am unable to connect to the simple game server using netcat though.

I followed the tutorial here:
https://agones.dev/site/docs/getting-started/create-gameserver/

I see the nodes, pods, and gameserver in the kubectl commands. Any ideas what could be the issue? Did I potentially do something incorrectly?

**Comment by markmandel** [comment]
Please fill out this part of our Bug template:

**Environment**:
- Agones version: 
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Install method (yaml/helm):
- Troubleshooting guide log(s):
- Others:

Otherwise we have no way of debugging the issue unfortunately.

**Comment by emrys90** [comment]
> Please fill out this part of our Bug template:
> 
> **Environment**:
> 
> * Agones version:
> * Kubernetes version (use `kubectl version`):
> * Cloud provider or hardware configuration:
> * Install method (yaml/helm):
> * Troubleshooting guide log(s):
> * Others:
> 
> Otherwise we have no way of debugging the issue unfortunately.

I install Agones and create the game server using the following commands:
kubectl create namespace agones-system
kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/release-1.20.0/install/yaml/install.yaml
kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/release-1.20.0/examples/simple-game-server/gameserver.yaml

So its Agones 1.20 using yaml.

My netcat call does this and freezes with no other data returned:
nc -v -u 45.79.165.106 7744
45.79.165.106 (45.79.165.106:7744) open

I've tried this on Kubernetes version 1.21 as recommended in the guide, and also on 1.22. This is running on Linode, and I've tried both a US east and Singapore regions (I'm currently traveling in Thailand if that matters for any connection issues).

I posted this help discussion on Linode where someone in their community was able to install it successfully. I've followed the exact same things as them from what I can tell, not sure what I am doing wrong.
https://www.linode.com/community/questions/22449/having-an-issue-connecting-to-a-server-running-in-kubernetes

Any help would be very much appreciated. I have a production game where I am currently just running X servers always online. I am trying to switch to this so we can reduce our server hosting costs, and also be able to scale past our normal peak server counts as needed without having players unable to connect.
